### PR TITLE
Added User Profile page

### LINKED ISSUE #18: Create User Profile Screen

### PR DESCRIPTION
**Pull request by vrajp13**
- Added a new user page
- Modified the navbar by including the option to go to the user page

### CODE REVIEW

**Comment by vrajp13** [comment]
- Minor UI changes on the user page

File: components/work-schedule/schedule-modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+      <div className="py-2 d-flex justify-content-around">
+        {
+          <>
+            <Button
+              variant="success"
+              className={styles.scheduleAddSubmitButton}
+              onClick={onButtonSelect}
+            >
+              Submit
```
Comment (ibaniya98) [inline components/work-schedule/schedule-modal/index.tsx]: Is there a different functionality for Submit and Update buttons?

**Review by ibaniya98 — COMMENTED** [review]
NIT: Just a design opinion. We have the name twice on the same screen.
image
Would it be possible to move the image inside the card and remove the "Bob" portion?
image

Or something like this for instance (don't need to background image since we already have one for the page) by removing the "name" and "email" title and directly listing the values only.
image

File: components/work-schedule/schedule-modal/index.tsx (hunk 1 above)
Comment (vrajp13) [reply components/work-schedule/schedule-modal/index.tsx]: That will be once I create the APIs for it, but right now its the same functionality

**Comment by vrajp13** [comment]
- Updated the user page with the design idea that Ishan suggested
- Also made the CSS elements in Home and User page responsive

**Review by ibaniya98 — DISMISSED** [review]
LGTM

**Comment by vrajp13** [comment]
This the new design now: 
image

**Review by cpc2820 — APPROVED** [review]
Looks good. We should merge the user page with the Home Screen in the next sprint though.

**Comment by vrajp13** [comment]
we are going to do that

**Pull request merged**